From 060ad28b9ca5e99d7c6e8010132da58a3a8a0585 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Fri, 22 Jan 2016 10:19:18 -0300
Subject: [PATCH] pqueue.h: include sys/types.h

We're using u_int32_t here, this breaks on musl libc builds otherwise.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
Patch status: applied upstream:
https://sourceforge.net/p/pptpclient/git/ci/ee8ef349d9543cb48045a27d1cdd801fc6e910e8/

 pqueue.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pqueue.h b/pqueue.h
index a56590f..8c60c62 100644
--- a/pqueue.h
+++ b/pqueue.h
@@ -3,6 +3,7 @@
 
 #include <time.h>
 #include <sys/time.h>
+#include <sys/types.h>
 
 /* wait this many seconds for missing packets before forgetting about them */
 #define DEFAULT_PACKET_TIMEOUT 0.3
-- 
2.4.10

